include/crocoddyl/core/activations/quadratic-barrier.hpp

```diff
@@ -2,6 +2,7 @@
 // BSD 3-Clause License
 //
 // Copyright (C) 2019-2020, LAAS-CNRS, University of Edinburgh
+// Copyright (C) 2021, University of Oxford
```

Comment:
Would you write this in the same line? (Same for the other files)

This is just to simplify the copyright signature. Indeed, I did this simplification a few PR ago.